App/arca/components/Sidebar.jsx

```diff
@@ -0,0 +1,114 @@
+import React from "react";
+import { useState } from "react";
+import { Link } from "react-router-dom";
+import "../assets/css/components.scss";
+import "../assets/css/helpCenter.scss";
+import "../assets/css/main.scss";
+import menuimg from "../assets/img/icons/menu-burger.svg";
+import arcalogo from "../assets/img/icons/logo.svg";
+import ItemsSidebar from "./ItemsSidebar";
+import dashboardimg from "../assets/img/icons/ico_apps-font_vio}.svg";
+import estetoscopioimg from "../assets/img/icons/ico_stethoscope.svg";
+import usersimg from "../assets/img/icons/users-alt-free-icon-font.svg";
+import documentimg from "../assets/img/icons/ico_document-signed.svg";
+import userprofile from "../assets/img/icons/ico_user-free-icon-font (1).svg";
+import help from "../assets/img/icons/help.svg";
+
+
+
+function Sidebar(props) {
+  const [estadoSidebar, setEstadoSidebar] = useState(false);
+  const [estadoMenu, setEstadoMenu] = useState(false);
+
+  const bodyClassName = estadoSidebar ? "body-dashboard body-dashboard-s" : "body-dashboard";
+  const btnSidebar = estadoSidebar ? "btn-des-sidebar btn-des-sidebar-s" : "btn-des-sidebar";
+  const imgBtn = estadoSidebar ? "img-btn img-btn-s" : "img-btn";
+  const imgSrc = estadoSidebar ? "../assets/img/icons/angle-double-small-right-free-icon-font.svg" : "../assets/img/icons/angle-double-small-left-free-icon-font.svg";
+  const arcaImg = estadoSidebar ? "arca-s" : "arca";
+  const conHelp = estadoSidebar ? "con-help-center con-help-center-s" : "con-help-center";
+
+
+  function click() {
+    setEstadoSidebar(!estadoSidebar);
+    { props.onClick() }
+  };
+
+  const bodyClassNameMenu = estadoMenu ? "body-dashboard body-dashboard-m" : "body-dashboard";
+  const btnMenu = estadoMenu ? "con-sidebar-a con-sidebar-m" : "con-sidebar-a";
+  const body = document.body;
+
+  function menu() {
+    if (estadoMenu) {
+      setEstadoMenu(false);
+      body.style.overflowY = "auto";
```

Comment:
You can change the styles almost the same as the `className` attribute but using `style`, instead of change the DOM in this way